packages/sn-client-utils/package.json

```
@@ -33,5 +33,7 @@
   "bugs": {
     "url": "https://github.com/SenseNet/sn-client/issues"
   },
-  "homepage": "https://sensenet.com"
-}
+  "homepage": "https://sensenet.com",
+  "dependencies": {
+    "tslib": "^1.9.3"
+  }
```

```suggestion
  }
}
```
A closing curly brace is missing from here